control-plane/tests/test_phase56_3_case_timeline_projection.py

```
@@ -0,0 +1,456 @@
+    EvidenceRecord,
+    ReconciliationRecord,
+    RecommendationRecord,
+    ServicePersistenceTestBase,
+    _approved_binding_hash,
+    _phase20_notify_identity_owner_payload,
+)
+
+
+class Phase563CaseTimelineProjectionTests(ServicePersistenceTestBase):
+    def test_case_detail_projects_required_segments_with_authority_posture(self) -> None:
+        store, service, promoted_case, evidence_id, reviewed_at = (
```

Ruff (0.15.12)

[warning] Unpacked variable `store` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)